### PR TITLE
Add Height and HeightRange option. 

### PR DESCRIPTION
**Pull request by masterworgen**
**Pull Request Description**

### Summary
I have added two new tags to the library, `<<Height number>>` and `<<HeightRange number>>`, which allow users to set the height for individual rows and a range of rows respectively.

### Changes Made
1. **`<<Height number>>` Tag**: This tag sets the height for a specific row. When this tag is processed, the row height is adjusted according to the specified number.
   
   **Example Usage**:
   ```
   <<Height 20>>
   ```
   After processing, the row containing this tag will have a height of 20 units.

2. **`<<HeightRange number>>` Tag**: This tag sets the height for a range of rows. When this tag is processed, the specified range of rows will have their heights set to the given number.
   
   **Example Usage**:
   ```
   <<HeightRange 30>>
   ```
   After processing, the rows in the specified range will each have a height of 30 units.

### Detailed Implementation
- The `<<Height number>>` tag parses the number and applies it to the height property of the targeted row.
- The `<<HeightRange number>>` tag parses the number and applies it to the height property of the rows within the specified range.

These enhancements aim to provide more flexibility and control over row formatting in the library.

Thank you for considering this pull request. I believe these additions will greatly benefit users by providing more precise control over row heights.

### CODE REVIEW

**Comment by Pankraty** [comment]
Please, add some tests - maybe something like [this](https://github.com/ClosedXML/ClosedXML.Report/blob/develop/tests/ClosedXML.Report.Tests/DeleteTagTests.cs) checking the new tags, or put a template with these tags into test/Templates folder and make a test like [this](https://github.com/ClosedXML/ClosedXML.Report/blob/develop/tests/ClosedXML.Report.Tests/XlTemplateTests.cs#L419).

Notice there are failing tests here.

**Comment by masterworgen** [comment]
Add and fix tests

File: tests/Templates/4D366000
Comment (Pankraty) [inline tests/Templates/4D366000:1]: It seems, this file was committed by mistake.

Comment (masterworgen) [reply tests/Templates/4D366000:1]: Removed

**Pull request merged**